projects/ngx-wig/src/lib/config.ts

```diff
@@ -22,43 +22,43 @@ export const DEFAULT_LIBRARY_BUTTONS: TButtonLibrary = {
     label: 'UL',
     title: 'Unordered List',
     command: 'insertunorderedlist',
-    styleClass: 'list-ul',
-    icon: 'icon-list-ul'
+    styleClass: 'nw-list-ul',
+    icon: 'nwicon-list-ul'
```

Comment:
Can we say `nw-icon-list-ul` ?

Reply:
As I commented above, the icon would receive the [style](https://github.com/stevermeister/ngx-wig/blob/61cb2a899a2d337be4650353252fea2c2914679b/projects/ngx-wig/src/lib/ngx-wig-component.css#L9) for `[class^="nw-"]`, which is not the case with the current library version. This selector applies to all class names with prefix `nw-`.
If you are fine with this, then let's set the style as you suggest.

Reply:
Let's see what @stevermeister has to say 🙂

Reply:
@kalski that's an excellent point! 

it's good to separate **styles for the editor** and **styles for content** (that you are editing inside)

could we make it `nw-` and `nwe-`?  to have a namespace prefix with "-" for both cases

Reply:
@bampakoa we made our comments almost simultaneously )))

Reply:
@stevermeister I agree the suggested naming is not the most clear one but I think the editor / content differentiation is a whole new story, not related to this PR. 

The initial problem was the too generic class names so I wanted to address particularly this issue without being intrusive to other library users.

Reply:
If you look closer at the screenshot you'll notice that even in **the editor** part, the icons parents were with class name `nw-button`, with the `nw-` prefix. This is so even in the current library versions. 
I am not sure of the reasoning and the correct scope for this style but possibly a new issue could be opened to sort this out.

Reply:
I think you got the point of the `nw-` styles, which would be applied to these elements, which will no longer have generic css class names. 
However, these elements are not purely editor or content related. And I don't want to dive into this separation here. 

Do you think this PR is ok to be merged ?

Reply:
If I get you right you want me to rename

`nwmenu-dropdown` -> `nwe-dropdown`
`nwicon` -> `nwe-icon`

Is this correct ? 

This will still be a good naming in terms of backwards compatiblity with the `nw-` style. 

Yet I think this style applicability to the elements should be separately reviewed.

Reply:
@stevermeister , can we move this forward ?

Reply:
@kalski sorry for keeping you waiting, I was away for a while. 

> nwmenu-dropdown -> nwe-dropdown
> nwicon -> nwe-icon

yes, please